project/profiles/capacity4more/behat/features/group_content_permissions.feature

```diff
@@ -14,109 +14,859 @@ Feature: Test creation of the content permissions.
+    Given  I am logged in as user "<user>"
+    When   I go to "<path>"
+    Then   I should not have access to the page
+
+    Examples:
+      | user        | path                 |
+      | badhairday  | /archived/node/add/photoalbum |
+      | isaacnewton | /archived/node/add/photoalbum |
+      | turing      | /archived/node/add/photoalbum |
+      | alfrednobel | /archived/node/add/photoalbum |
+      | badhairday  | /archived/node/add/photo |
+      | isaacnewton | /archived/node/add/photo |
+      | turing      | /archived/node/add/photo |
+      | alfrednobel | /archived/node/add/photo |
+      | badhairday  | /archived/node/add/event |
+      | isaacnewton | /archived/node/add/event |
+      | turing      | /archived/node/add/event |
+      | alfrednobel | /archived/node/add/event |
+      | badhairday  | /archived/node/add/document |
+      | isaacnewton | /archived/node/add/document |
+      | turing      | /archived/node/add/document |
+      | alfrednobel | /archived/node/add/document |
+      | badhairday  | /archived/node/add/discussion |
+      | isaacnewton | /archived/node/add/discussion |
+      | turing      | /archived/node/add/discussion |
+      | alfrednobel | /archived/node/add/discussion |
+      | badhairday  | /archived/node/add/wiki-page |
+      | isaacnewton | /archived/node/add/wiki-page |
+      | turing      | /archived/node/add/wiki-page |
+      | alfrednobel | /archived/node/add/wiki-page |
 
   @api
-  Scenario: Group non member should not have access to group 'create photo' form.
-    Given I am logged in as user "badhairday"
-    When  I go to "movie-corner/node/add/photo"
-    Then  I should not have access to the page
+  Scenario: Restore "Archived group" group state to 'Public'.
+    Given  I am logged in as user "mariecurie"
+    When   I change access of group "Archived group" to "Public"
+    And    I am an anonymous user
+
+  @api
+  Scenario: Set "Deleted group" group state to 'Public'. Anonymous user asked to log in.
+    Given I am logged in as user "mariecurie"
+    When  I change access of group "Deleted group" to "Public"
+    Then  I am an anonymous user
+    And   I go to "deleted"
+    And   I should see "Please log in"
 
   @api
-  Scenario: Group member should have access to group 'create photo' form.
-    Given I am logged in as user "isaacnewton"
-    When  I go to "movie-corner/node/add/photo"
+  Scenario Outline: As site admin, access create content form, with purl prefix of deleted public group.
+    Given I am logged in as user "<user>"
+    When  I go to "<path>"
     Then  I should have access to the page
 
+    Examples:
+      | user        | path                 |
+      | mariecurie  |  /deleted/node/add/photoalbum |
+      | mariecurie  |  /deleted/node/add/photo |
+      | mariecurie  |  /deleted/node/add/event |
+      | mariecurie  |  /deleted/node/add/document |
+      | mariecurie  |  /deleted/node/add/discussion |
+      | mariecurie  |  /deleted/node/add/wiki-page |
+
```

Comment:
no we agreed with Aki this AM that both pending and deleted groups would prevent any user from creating content - so please add lines 716 to 721 in the scenario below, described under line 724